doc/Getting-started/index.md

```diff
@@ -53,24 +53,17 @@ The current version of Kaa Sandbox supports Oracle VirtualBox version 5.1.2 and
 
 4. After you loaded the image, open the **Oracle VM VirtualBox Manager**, click **Settings** > **System** and adjust the amount of RAM and processors available to the VM.
 For optimal performance, we recommend at least 4 GB of RAM and at least 2 CPUs.
+Apply your settings and start the virtual machine.
 
-	>**TIP:** Configure your network settings.  
-	>The default VM networking configuration is NAT, which enables your host machine to connect to the [Kaa server]({{root_url}}Glossary/#kaa-server).
-	>This means that you will need to deploy your [Kaa client]({{root_url}}Glossary/#kaa-client) applications to your host machine to enable their communication with the Sandbox server.
-	>
-	>You can deploy the Kaa client applications to any hardware other than your host machine, and access Kaa web interfaces from your network devices.
+	>**NOTE:** Your instance of Kaa Sandbox works out of the box with the default VM networking configuration set to NAT.
+	>This enables your host machine to connect to the [Kaa server]({{root_url}}Glossary/#kaa-server).
```

Comment:
As far as I know, in current configuration this allows connecting from any machine, not only host.